components/Detail.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from 'react'
+import styled from 'styled-components/native'
+import { SafeAreaView, ScrollView, StyleSheet } from 'react-native'
+
+export const Detail = () => {
+  const [info, setInfo] = useState([])
+
+  useEffect(() => {
+    fetch('https://api.nasa.gov/planetary/apod?api_key=qSlGOjPnCXc7pgWt5QcdsI3nw45FfocwP6EggaOZ')
+      .then((res) => res.json())
+      .then((json) => setInfo(json))
+  }, [])
```

Comment:
it looks like this is exactly the same fetch as in the other component.
So I would suggest you do the useEffect in the parent component and pass along  or at least just saving the info that you need in the state variables instead of the whole thing.
So here,  if you only care about the explanation, just save the explanation:
```
const [info, setInfo] = useState('')
and
.then((json) => setInfo(json.explanation))
```